pkg/authorization/registry/resourceaccessreview/rest.go

```diff
@@ -32,29 +33,53 @@ func (r *REST) New() runtime.Object {
 func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
 	resourceAccessReview, ok := obj.(*authorizationapi.ResourceAccessReview)
 	if !ok {
-		return nil, errors.NewBadRequest(fmt.Sprintf("not a resourceAccessReview: %#v", obj))
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a resourceAccessReview: %#v", obj))
 	}
 	if err := kutilerrors.NewAggregate(authorizationvalidation.ValidateResourceAccessReview(resourceAccessReview)); err != nil {
 		return nil, err
 	}
-
-	namespace := kapi.NamespaceValue(ctx)
-
-	attributes := &authorizer.DefaultAuthorizationAttributes{
-		Verb:     resourceAccessReview.Verb,
-		Resource: resourceAccessReview.Resource,
+	// if a namespace is present on the request, then the namespace on the SAR must be empty.  In such a case,
```

Comment:
I don't see the code requiring the namespace on the RAR to be empty... can the comment just be abbreviated to "if a namespace is present on the request, the namespace on the RAR is overwritten..."?